polls/imdbScraper.py

```diff
@@ -0,0 +1,123 @@
+import logging
+import sys
+import traceback
+
+import requests
+
+from bs4 import BeautifulSoup
+
+
+def get_search_results(movie):
+
+    url = "http://www.imdb.com/find?s=tt&q=" + movie
+
+    try:
+        r = requests.get(url)
+    except requests.exceptions.RequestException as e:
+        logging.info(e)
+        traceback.print_exc(e)
+        json_object = {}
+        json_object['error'] = "Service unavailable"
+        return json_object
```

Comment:
Status code has been handled in views. Please turn off wifi and then try searching any movie.